conf/evolutions/default/98.sql

```diff
@@ -2,6 +2,12 @@
 
 ALTER TABLE `patient_encounters`
 ADD COLUMN `is_diabetes_screened` BOOL NULL DEFAULT NULL AFTER `mission_trip_id`;
+ALTER TABLE `patient_encounters`
```

Comment:
@msaleh97 old evolution files shouldn't be changed - go ahead and add a file 105.sql in this case and put the changes there.

If you change old evolutions the database will downrev to 98.sql and try to rebuild itself